java-core/google-cloud-core/src/main/java/com/google/cloud/testing/BaseEmulatorHelper.java

```diff
@@ -112,14 +115,21 @@ protected final void startProcess(String blockUntilOutput)
     }
   }
 
+  /** This method is obsolete. Use {@link #waitForProcessDuration(java.time.Duration)} instead */
+  @ObsoleteApi("Use waitForProcessDuration() instead")
+  protected final int waitForProcess(org.threeten.bp.Duration timeout)
+      throws IOException, InterruptedException, TimeoutException {
+    return waitForProcessDuration(toJavaTimeDuration(timeout));
+  }
```

Comment:
Can we double check that this overload is needed? I believe it's protected and final and subclasses shouldn't be able to override this method. I *think* if we just migrate all internal uses cases in this repo to use java.time.Duration, we should be good.

Reply:
I wrote this on the wrong one.